xbmc/filesystem/DllLibCurl.cpp

```diff
@@ -174,6 +196,29 @@ void DllLibCurlGlobal::CheckIdle()
 #endif
 }
 
+CCriticalSection DllLibCurlGlobal::m_curlLock;
+
+void DllLibCurlGlobal::lock(CURL_HANDLE *handle, curl_lock_data data, curl_lock_access access, void *useptr ) {
+  (void)handle;
+  (void)access;
```

Comment:
why?

Reply:
This was a C&P from a sample. Don't think it is required. -> Removed